semgrep/tests/conftest.py

```diff
@@ -220,3 +221,12 @@ def pytest_configure(config):
 def pytest_runtest_setup(item):
     if item.get_closest_marker("qa") and not item.config.getoption("--qa"):
         pytest.skip("skipping QA tests, add --qa flag to run them")
+
+
+# Add `public_repo_url` as a parameter to your test to use this. It generates 1 test case per repo url in `ALL_REPOS`
+def pytest_generate_tests(metafunc):
```

Comment:
Can we simplify this to just have two different funtions: one that runs the 5 repos and one that runs a lot of repos. Not sure why this needs to be a fixture and have a separate file with all the repos? Seems like a lot of complexity.

Reply:
Discussed offline. We can revisit re: complexity when it becomes a problem merging as is.